test/services/notifications/process-returned-letter.service.test.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+const { generateUUID, timestampForPostgres, today } = require('../../../app/lib/general.lib.js')
+
+// Test helpers
+const EventHelper = require('../../support/helpers/event.helper.js')
+const NotificationHelper = require('../../support/helpers/notification.helper.js')
+const NotificationModel = require('../../../app/models/notification.model.js')
+
+// Thing under test
+const ProcessReturnedLetterService = require('../../../app/services/notifications/process-returned-letter.service.js')
+
+describe('Submit Returned Letter Service', () => {
```

Comment:
Named after previous service name.


```suggestion
describe('Notifications - Process Returned Letter service', () => {
```